lib/gemstash/migrations/06_compact_index.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    alter_table :versions do # TODO: backfill info_checksum, sha256, required_ruby_version, required_rubygems_version
```

Comment:
@indirect any thoughts on how best to do this, now that the PR is passing?

Reply:
Should we add a `gemstash update` or `gemstash sync` command that would be a standard idempotent way to reindex/update after a new version is released?

Have we done a migration like this before?

Reply:
yeah, I think a command to `migrate` or `update` is the way to go, with some docs about how to update to the newer version.

Reply:
Are we sure we need an explicit command? I found `Gemstash::Env.migrate` which handles running migrations. That is called in `Gemstash::Env.db`, so basically anytime the db is accessed it is automatically migrated before rturning a connection.